src/components/GeneralInfo/SystemStatusCard/SystemStatusCard.js

```diff
@@ -43,6 +44,25 @@ const SystemStatusCardCore = ({
         ? [
             {
               title: 'Last seen',
+              value: entity && (
+                <DateFormat
+                  date={
+                    entity.per_reporter_staleness &&
+                    Object.values(entity.per_reporter_staleness)
+                      .map((reporter) => reporter.last_check_in)
+                      .sort()
+                      .reverse()[0]
+                  }
+                  type="exact"
+                />
+              ),
+            },
+          ]
+        : []),
+      ...(hasLastUpdated
+        ? [
+            {
+              title: 'Last updated',
```

Comment:
Please change `entity.per_reporter_staleness &&` to `entity.per_reporter_staleness ?` and have the false option display "Not available". Do this for both "Last seen" and "Last updated".

Also, let's write a couple of tests in the `SystemStatusCard.tests.js` file.

- One that specifically tests the map and sort functionality
- One that tests that an invalid date properly displays something like, "Not available"